region/crateria/west/Gauntlet Energy Tank Room.json

```diff
@@ -778,9 +821,24 @@
     },
     {
       "link": [4, 3],
-      "name": "Base",
+      "name": "Power Bombs",
       "requires": [
-        "h_canBombThings"
+        "h_canUsePowerBombs",
+        {"or": [
+          "canCarefulJump",
+          {"acidFrames": 5}
+        ]}
+      ]
+    },
+    {
+      "link": [4, 3],
+      "name": "Bombs",
+      "requires": [
+        "h_canBombThings",
+        {"or": [
+          "canInsaneJump",
+          {"acidFrames": 20}
```

Comment:
I would think a tricky jump would place the bomb and return to the left (freely), then go back in and place the second. Spring helps a bit too.

I think i would go: 
10 frames or spring plus ...
- 200 frames
- 100-150 + careful
- 10-20 + tricky

then either insane on its own strat, or added as an option with the spring as well